### PR TITLE
docs for EBS provisioning

### PR DESCRIPTION
**Pull request by puja108**
RFR

### CODE REVIEW

File: content/guides/using-persistent-volumes-on-aws/index.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+date = "2017-06-26"
+type = "page"
+weight = 50
+tags = ["guide"]
++++
+
+# Using Persistent Volumes on AWS
+
+If your cluster is running in the cloud on Amazon Web Services (AWS), it comes with a dynamic storage provisioner for Elastic Block Storage (EBS).
+
```
Comment (marians) [inline content/guides/using-persistent-volumes-on-aws/index.md]: How about another sentence on the business value? Like "This enables you to store data beyond the lifetime of a single worker node." (not sure if that is factually right)

File: content/guides/using-persistent-volumes-on-aws/index.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+## Storage Classes
+
+Your Kubernetes cluster will have a default Storage Class `gp2` deployed, which will automatically get selected if you do not specify the Storage Class in your Persistent Volumes.
+
```
Comment (marians) [inline content/guides/using-persistent-volumes-on-aws/index.md]: Does `gp2` stand for anything? If yes, let's explain here.

File: content/guides/using-persistent-volumes-on-aws/index.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+As a Cluster Admin you can create additional Storage Classes or edit the default class to use different types of EBS or for example add encryption.
+
+For this you just need to create (or edit) [Storage Class objects](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#aws).
+
+## Creating Persistent Volumes
+
+To create a Persistent Volume (PV) you just need to create a [Persistent Volume object](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistent-volumes). This PV then needs to be claimed using a Persistent Volume Claim (PVC).
+
```
Comment (marians) [inline content/guides/using-persistent-volumes-on-aws/index.md]: How about a YAML example here?

File: content/guides/using-persistent-volumes-on-aws/index.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+Alternatively, you can directly create a [Persistent Volume Claim object](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims), which will automatically create a corresponding Persistent Volume for you.
+
```
Comment (marians) [inline content/guides/using-persistent-volumes-on-aws/index.md]: Again, YAML example

File: content/guides/using-persistent-volumes-on-aws/index.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+Under the hood, the Dynamic Storage Provisioner will take care that a corresponding EBS Volume with the correct parameters is created.
+
+The EBS Volume and its data will persist as long as the corresponding PV is not deleted from your Kubernetes cluster.
```
Comment (marians) [inline content/guides/using-persistent-volumes-on-aws/index.md]: Can we put that in a positive phrase?

`The EBS Volume and its data will persist as long as the corresponding PV resource exists. Deleting the resource will also delete the corresponding EBS volume, which means that all stored data will be lost at that point.`

File: content/guides/using-persistent-volumes-on-aws/index.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+## Using Persistent Volumes in a Pod
+
+Once you have a Persistent Volume Claim you can [claim it as a Volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#claims-as-volumes) in your Pods.
+
```
Comment (marians) [inline content/guides/using-persistent-volumes-on-aws/index.md]: What are we using from the Pod? The persistent volume? Or the persistent volume claim? The headline seems to contradict the sentence in line 34.

File: content/guides/using-persistent-volumes-on-aws/index.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+Note that an EBS Volume can only be used by a single Pod at the same time.
```
Comment (marians) [inline content/guides/using-persistent-volumes-on-aws/index.md]: What will happen if I don't?

File: content/guides/using-persistent-volumes-on-aws/index.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+
+Under the hood the EBS Volume stays detached from your nodes as long as it is not claimed by a Pod. As soon as a Pod claims it, it gets attached to the node that holds the Pod.
+
+## Example
+
+First we create a PVC:
+
+```
```
Comment (marians) [inline content/guides/using-persistent-volumes-on-aws/index.md]: If you add `yaml` to the three backticks, you get enforce syntax highlighting.

File: content/guides/using-persistent-volumes-on-aws/index.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+      storage: 6Gi
+```
+
+Note that the Access Mode while being fixed with EBS still needs to be defined as `ReadWriteOnce` in the manifest.
+
+Further, as we are not defining a Storage Class the Kubernetes cluster will just take the default storage class (here `gp2`).
+
+Now we can create a Pod that uses our PVC:
+
+```
```
Comment (marians) [inline content/guides/using-persistent-volumes-on-aws/index.md]: `yaml`

File: content/guides/using-persistent-volumes-on-aws/index.md (hunk 3 above)
Comment (puja108) [inline content/guides/using-persistent-volumes-on-aws/index.md]: I wouldn't want to encourage this, should we maybe delete this then?

File: content/guides/using-persistent-volumes-on-aws/index.md (hunk 2 above)
Comment (puja108) [inline content/guides/using-persistent-volumes-on-aws/index.md]: it's an AWS EBS type just like `m3.large`

File: content/guides/using-persistent-volumes-on-aws/index.md (hunk 4 above)
Comment (puja108) [inline content/guides/using-persistent-volumes-on-aws/index.md]: see below in the example section

File: content/guides/using-persistent-volumes-on-aws/index.md (hunk 6 above)
Comment (puja108) [inline content/guides/using-persistent-volumes-on-aws/index.md]: you actually claim a persistent volume claim...

Comment (puja108) [inline content/guides/using-persistent-volumes-on-aws/index.md]: but that uses the persistent volume which in turn uses EBS

File: content/guides/using-persistent-volumes-on-aws/index.md (hunk 7 above)
Comment (puja108) [inline content/guides/using-persistent-volumes-on-aws/index.md]: you just cannot, I will add this to the note

File: content/guides/using-persistent-volumes-on-aws/index.md (hunk 1 above)
Comment (puja108) [inline content/guides/using-persistent-volumes-on-aws/index.md]: done

File: content/guides/using-persistent-volumes-on-aws/index.md (hunk 3 above)
Comment (puja108) [inline content/guides/using-persistent-volumes-on-aws/index.md]: switching

**Comment by puja108** [comment]
@marians you wanna check again?

**Pull request merged**